src/containers/List/ListWrapper.tsx

```diff
@@ -14,12 +14,12 @@ import { Link } from 'react-router-dom'
 import ApplicationsList from '../../components/List/ApplicationsList'
 import PaginationBar from '../../components/List/Pagination'
 import ListFilters from './ListFilters/ListFilters'
-import { useApplicationFilters } from '../../utils/data/applicationFilters'
+import { useGetFilterDefinitions } from '../../utils/helpers/list/useGetFilterDefinitions'
```

Comment:
Tiny -- but there's a blank link in the middle of the imports (line 3). Might want to remove.